src/select/select.js

```diff
@@ -457,6 +467,16 @@ class Select extends React.Component<PropsT, StatelessStateT> {
       ),
     };
   }
+
+  getAccessibilityProps() {
```

Comment:
shouldn't we address these in a separate PR?

Reply:
https://github.com/uber-web/baseui/pull/409